asf_lifetime_cost_model/config/base.yaml

```
@@ -0,0 +1,61 @@
+# List of possible arguments for a variety of parameters and default values
+#that enable the calculation of the lifetime cost of different heating systems.
+
+ashp_efficiency_options: [
+    "reference", # SCOP 3
+    "high", # SCOP > 3
+    "low", # SCOP < 3
+  ]
+
+boiler_efficiency_default: 0.85 # boiler efficiency
+
+life_span_default: # in years
+  ashp: 15 # years
+  boiler: 15 # years
+
+maintenance_costs_default:
+  ashp:
+    annual_cost: 150 # GBP
+    number_of_years: 5 # HP warranty
+  boiler:
+    annual_cost: 80 # GBP
+    number_of_years: 15 # boiler lifetime
+
+ashp_subsidy_options:
+  [
+    "flat",
+    "slow stepdown",
+    "fast stepdown",
+    "high",
+    "zero from 2028",
+    "smallest",
+    "no subsidy",
+  ]
+
+ashp_market_cost_change: [
+    { "name": "optimistic", "annual_price_reduction": 0.045 }, # 4.5% annual price reduction
+    { "name": "moderate", "annual_price_reduction": 0.025 }, # 2.5% annual price reduction
+    { "name": "conservative", "annual_price_reduction": 0.01 }, # 1% annual price reduction
+  ]
+
+ashp_loans_options: [True, False] # whether ASHPs are purchased with loans
+loan_interest_rate_default: 0.05 # percent
+
+fossil_fuel_prices_options: [
+    "reference", # DESNZ_Ref
+    "low", # DESNZ_Low
+    "high", # DESNZ_High
+  ]
```

there are more scenarios in the DESNZ workbook that we might want to use, so I think these should be more aligned to how they are named in the original data source!
```suggestion
wholesale_prices_projection_options: [
    "reference", # DESNZ_Ref
    "low fossil fuel prices", # DESNZ_Low
    "high fossil fuel prices", # DESNZ_High
  ]
```

---

love this, thank you @eglucas! we're so lucky to have your knowledge of these datasets.

Are there any other variables that you think we should rename?

---

i think this was the only one that wasn't very clear - all the rest make sense to me 👍🏼